test-suite/test-cases/rdfa1.1/html5/0325.sparql

```diff
@@ -1,7 +1,10 @@
 BASE <http://example.org/>
 PREFIX schema: <http://schema.org/>
 ASK WHERE {
-  #foo schema:refers-to _:p .
-  #bar schema:refers-to _:p .
-  _:p schema:name "Amanda" .
+  #foo schema:refers-to _:p ;
+  	 a schema: .
+  #bar schema:refers-to _:p ;
+  	a schema: .
+  _:p schema:name "Amanda" ;
```

Comment:
Note that `#foo` and `#bar` are commented out lines, so the addition of `a schema:` does not parse. They should be `<#foo>` and `<#bar>`.

The type is not generated because `@typeof` is a [TERMorCURIEorAbsIRIs](https://www.w3.org/TR/rdfa-core/#dfn-termorcurieorabsiris), which is a whitespace separate list of [TERMorCURIEorAbsIRI](https://www.w3.org/TR/rdfa-core/#dfn-termorcurieorabsiri), so it results in an empty list. `@typeof=""` is often used to create a new blank node subject.